### ISSUE TITLE
Subscription process is unnecessarily verbose

### ISSUE THREAD

**Issue by pwmcintyre** [integrations/slack#1351]
**Is your feature request related to a problem? Please describe.**

Yes; the process of subscribing is unnecessarily verbose
1. the slash command is public
2. the response to the command is public

This is especially annoying if you're going through the process of subscribing to many "features" and "repos"

**Describe the solution you'd like**

Both the command and the response should be private

**Additional context**

![image](https://user-images.githubusercontent.com/774725/152460171-8f8e0450-bba5-4847-a4a1-10ab8cb02d3d.png)

**Comment by gregmarr** [comment]
I agree that the command does not need to be echoed, but having a record in the channel of who configured the integration can be helpful.

**Comment by jamesastound** [comment]
Would be nice if you could add a special character preceding the command `/github !unsubscribe ...` to disable the echo feature.

**Comment by nicolas-serbin** [comment]
Any news? Would be great to do this
